portal/app/[locale]/staking-dashboard/_components/lockup/index.tsx

```diff
@@ -87,13 +89,30 @@ export function getNearestValidValues({
 
 const Divider = () => <div className="h-px w-full bg-neutral-300/55" />
 
-const InfoRow = ({ label, value }: { label: string; value: string }) => (
+const InfoRow = ({ label, value }: { label: string; value: ReactNode }) => (
   <p className="flex justify-between text-sm font-medium text-neutral-600">
     <span>{label}</span>
     <span className="text-neutral-950">{value}</span>
   </p>
 )
 
+const VotingPowerEquivalence = ({
+  amount,
+  token,
+  votingPower,
+}: {
+  amount: string
+  token: EvmToken
+  votingPower: string
+}) => (
+  <div className="flex items-center gap-x-1">
+    <DisplayAmount amount={amount} showSymbol={true} token={token} />
+    <span className="text-neutral-600">=</span>
+    <DisplayAmount amount={votingPower} showSymbol={false} token={token} />
+    <span className="text-neutral-600">ve{token.symbol}</span>
```

Comment:
The voting power will always be "veHEMI". Let's the component display the token symbol.

```suggestion
    <DisplayAmount amount={votingPower} showSymbol={true} token={token} />
```